src/unix/lwt_unix.cppo.mli

```diff
@@ -409,6 +409,11 @@ sig
   (** [is_empty vs] is [true] if and only if [vs] has no I/O vectors, or all I/O
       vectors in [vs] have zero bytes. *)
 
+  val length : t -> int
+  (** [length vs] is the total length of the I/O vectors of [t]. In other words,
+      it is the sum of all the lengths of I/O vectors that have been appended
+      minus the sum of all the bits that have been dropped. *)
```

Comment:
The part about dropping seems confusing. Why not just say it's the number of bytes in the I/O vectors in `t`? It seems that the `drop` docs are sufficient to know that bytes can be dropped from the front of `t`, and so won't be counted here. I think the user can assume that an API is logically consistent with itself :)